guides/v2.2/cloud/cdn/cloud-fastly-custom-response.md

```diff
@@ -4,9 +4,6 @@ title: Customize error pages
 redirect_from:
    - /guides/v2.2/cloud/configure/fastly-vcl-extend-timeout.html
    - /guides/v2.3/cloud/configure/fastly-vcl-extend-timeout.html
```

Comment:
Do we want to redirect from 2.3 down to 2.2?

Reply:
I assume that this is a symlinked topic. So, in fact, both redirected links 2.2 and 2.3 will go to 2.3.

Reply:
Right. With symlinked files and multiple redirects, bottom one wins.